.eslintrc

```diff
@@ -30,8 +30,12 @@
     "react/jsx-one-expression-per-line": "off",
     "react/jsx-props-no-spreading": "off",
     "react/jsx-wrap-multilines": "off",
-    "react/function-component-definition": [2, { "namedComponents": "arrow-function" }],
+    "react/function-component-definition": [
+      2,
+      { "namedComponents": "arrow-function" }
+    ],
     "react/react-in-jsx-scope": "off",
-    "semi": "off"
+    "semi": "off",
+    "no-underscore-dangle": "off"
```

Comment:
Everyone cool with this? It allows us to use `_varname` which is helpful when defining temp vars inside of an effect, that share the same name as the var used w/ useState()

Reply:
Cool 👍